config/schemas/6/rapidast_schema.json

```diff
@@ -0,0 +1,605 @@
+                                                    "name": {
+                                                        "type": "string"
+                                                    },
+                                                    "value": {
+                                                        "type": "string"
+                                                    }
+                                                },
+                                                "required": [
+                                                    "name",
+                                                    "value"
+                                                ]
+                                            }
+                                        }
+                                    }
+                                },
+                                {
+                                    "if": {
+                                        "properties": {
+                                            "type": {
+                                                "const": "browser"
+                                            }
+                                        }
+                                    },
+                                    "then": {
+                                        "properties": {
+                                            "parameters": {
+                                                "properties": {
+                                                    "username": {
+                                                        "type": "string"
+                                                    },
+                                                    "password": {
+                                                        "type": "string"
+                                                    },
+                                                    "loginPageUrl": {
+                                                        "type": "string"
+                                                    },
+                                                    "loginPageWait": {
+                                                        "type": "string"
+                                                    },
+                                                    "verifyUrl": {
+                                                        "type": "string"
+                                                    },
+                                                    "loggedInRegex": {
+                                                        "type": "string"
+                                                    },
+                                                    "loggedOutRegex": {
+                                                        "type": "string"
+                                                    }
+                                                },
+                                                "required": [
```

Comment:
required parameters for `browser` authentication

Reply:
lgtm